client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -355,22 +355,24 @@ export class PluginsList extends Component {
 
 		const isJetpackIncluded = selectedPlugins.some( ( { slug } ) => slug === 'jetpack' );
 
-		const translationArgs = {
-			args: { pluginsCount },
-			count: pluginsCount,
-		};
-
 		const dialogOptions = {
 			additionalClassNames: 'plugins__confirmation-modal',
+			...( actionName === 'remove' && { isScary: true } ),
 		};
 
+		let pluginName;
+		const hasOnePlugin = pluginsCount === 1;
+
+		if ( hasOnePlugin ) {
+			const [ { name, slug } ] = selectedPlugins;
+			pluginName = name || slug;
```

Comment:
Are there cases when the plugin name is missing?

Reply:
Not sure, it was being done in the other place when we get the `pluginName`, so did the same thing here.